packages/framework/ini-files/config-specs.ini

```diff
@@ -2094,3 +2267,89 @@ opt-set-cmake-var Zoltan2_pamgenMeshAdapterTest_scotch_MPI_4_DISABLE BOOL FORCE
+opt-set-cmake-var TPL_BLAS_LIBRARY_DIRS STRING FORCE : ${BLAS_ROOT|ENV}/lib
+opt-set-cmake-var TPL_BoostLib_LIBRARIES STRING FORCE : ${BOOST_LIB|ENV}/libboost_program_options.a;${BOOST_LIB|ENV}/libboost_system.a
+opt-set-cmake-var TPL_Boost_LIBRARIES STRING FORCE : ${BOOST_LIB|ENV}/libboost_program_options.a;${BOOST_LIB|ENV}/libboost_system.a
+opt-set-cmake-var TPL_DLlib_LIBRARIES FILEPATH FORCE : "-ldl"
+opt-set-cmake-var TPL_HDF5_LIBRARIES STRING FORCE : ${HDF5_LIB|ENV}/libhdf5_hl.so;${HDF5_LIB|ENV}/libhdf5.a;${ZLIB_LIB|ENV}/libz.a;-ldl
+opt-set-cmake-var TPL_LAPACK_LIBRARIES STRING FORCE : -L${BLAS_ROOT|ENV}/lib;-lopenblas;-lgfortran;-lgomp
+opt-set-cmake-var TPL_LAPACK_LIBRARY_DIRS STRING FORCE : ${BLAS_ROOT|ENV}/lib
+opt-set-cmake-var TPL_METIS_LIBRARIES STRING FORCE : ${METIS_LIB|ENV}/libmetis.so
+opt-set-cmake-var TPL_Netcdf_LIBRARIES STRING FORCE : -L${NETCDF_C_ROOT|ENV}/lib64;${NETCDF_C_ROOT|ENV}/lib/libnetcdf.a;${PARALLEL_NETCDF_ROOT|ENV}/lib/libpnetcdf.a;${TPL_HDF5_LIBRARIES|CMAKE}
+
+#TPL_[INCLUDE|LIBRARY]_DIRS
+opt-set-cmake-var Netcdf_INCLUDE_DIRS STRING FORCE : ${NETCDF_C_INC|ENV}
+opt-set-cmake-var ParMETIS_INCLUDE_DIRS STRING FORCE : ${PARMETIS_INC|ENV}
+opt-set-cmake-var ParMETIS_LIBRARY_DIRS STRING FORCE : ${PARMETIS_LIB|ENV}
+opt-set-cmake-var Scotch_INCLUDE_DIRS STRING FORCE : ${SCOTCH_INC|ENV}
+opt-set-cmake-var Scotch_LIBRARY_DIRS STRING FORCE : ${SCOTCH_LIB|ENV}
+opt-set-cmake-var SuperLU_INCLUDE_DIRS STRING FORCE : ${SUPERLU_INC|ENV}
+opt-set-cmake-var SuperLU_LIBRARY_DIRS STRING FORCE : ${SUPERLU_LIB|ENV}
+
+#CXX Settings
+opt-set-cmake-var CMAKE_CXX_STANDARD STRING FORCE : 14
+opt-set-cmake-var CMAKE_CXX_FLAGS STRING : -fPIC -Wall -Warray-bounds -Wchar-subscripts -Wcomment -Wenum-compare -Wformat -Wuninitialized -Wmaybe-uninitialized -Wmain -Wnarrowing -Wnonnull -Wparentheses -Wreorder -Wreturn-type -Wsign-compare -Wsequence-point -Wtrigraphs -Wunused-function -Wunused-but-set-variable -Wunused-variable -Wwrite-strings
+
+#Package Options
+opt-set-cmake-var EpetraExt_ENABLE_HDF5 BOOL FORCE : OFF
+opt-set-cmake-var KOKKOS_ARCH STRING FORCE : VOLTA70
+opt-set-cmake-var KOKKOS_ENABLE_DEBUG BOOL FORCE : ON
+opt-set-cmake-var Kokkos_ENABLE_CUDA BOOL FORCE : ON
+opt-set-cmake-var Kokkos_ENABLE_CUDA_LAMBDA BOOL FORCE : ON
+opt-set-cmake-var Kokkos_ENABLE_CUDA_RELOCATABLE_DEVICE_CODE BOOL FORCE : OFF
+opt-set-cmake-var Kokkos_ENABLE_CXX11_DISPATCH_LAMBDA BOOL FORCE : ON
+opt-set-cmake-var Kokkos_ENABLE_Debug_Bounds_Check BOOL FORCE : ON
+opt-set-cmake-var MPI_EXEC_POST_NUMPROCS_FLAGS STRING FORCE : "-map-by;socket:PE=4"
+opt-set-cmake-var Panzer_FADTYPE STRING FORCE : "Sacado::Fad::DFad<RealType>"
+opt-set-cmake-var Phalanx_KOKKOS_DEVICE_TYPE STRING FORCE : CUDA
+opt-set-cmake-var Sacado_ENABLE_HIERARCHICAL_DFAD BOOL FORCE : ON
+opt-set-cmake-var Tpetra_INST_SERIAL BOOL FORCE : ON
+opt-set-cmake-var Zoltan_ENABLE_Scotch BOOL FORCE : OFF
+
+#Trilinos options
+opt-set-cmake-var Trilinos_ENABLE_DEBUG_SYMBOLS BOOL FORCE : OFF
```

Comment:
Is this necessary (is Trilinos_ENABLE_DEBUG_SYMBOLS set to ON via `use BUILD-TYPE|RELEASE`)?